tests/common/plugins/loganalyzer/loganalyzer.py

```diff
@@ -84,6 +88,11 @@ def _verify_log(self, result):
         if self.expect_regex and result["total"]["expected_match"] == 0:
             raise LogAnalyzerError(result)
 
+        # if the number of expected matches is provided
+        if (self.expect_regex and (self.expected_matches_target > 0)
```

Comment:
Where does this `expected_matches_target` variable get set? I see in the constructor but it's just set to 0.

Reply:
I haven't provided a separate method for this. using obj.var to set it now. Just followed the same approach used for setting regex's as well